### ISSUE TITLE
How to get data in dev server?

### ISSUE THREAD

**Issue by ChillarAnand** [pytube/pytube#4]
There is `data` dir which has all info. But when dev server is started, it is not showing up?

**Comment by logston** [comment]
Hey @ChillarAnand, I'm not sure I understand the issue. Can you rephrase?

**Comment by ChillarAnand** [comment]
I have cloned the repo and started server with `make serve`. When i open categories page in browser it is empty.

![p](https://cloud.githubusercontent.com/assets/4463796/15019907/7d5b56c6-123d-11e6-9dda-d07fd9fa2f58.png)

Can you tell me how to populate data into that page?

**Comment by logston** [comment]
Ah, I see. You have to run these commands:


This command will populate the content dir with `.rst` files:

```sh
$ make content
```

... and then this command will convert those `.rst` files into html:

```sh
$ make html
```

Finally, you will be able to serve those html files with:

```sh
$ make serve
```

**Comment by logston** [comment]
Running the first two commands above and take a while given the amount of data in the `data/` dir, FYI.

**Comment by ChillarAnand** [comment]
I tried running `make content` and it is throwing this error.


```py
# anand at anand in ~/sandbox/pytube on git:feature o [0:16:33]
$ make content
python bin/article_maker.py
Feeds generated without SITEURL set properly may not be valid
Traceback (most recent call last):
  File "bin/article_maker.py", line 391, in <module>
    main()
  File "bin/article_maker.py", line 387, in main
    run_article_maker_pool(args.process_count, verbose=args.verbose)
  File "bin/article_maker.py", line 334, in run_article_maker_pool
    json_file_paths = glob.iglob(pattern, recursive=True)
TypeError: iglob() got an unexpected keyword argument 'recursive'
make: *** [content] Error 1
```

**Comment by ChillarAnand** [comment]
Ah, my venv got corrupted.  Working fine now.

**Comment by logston** [comment]
👍
